src/index.js

```diff
@@ -5,107 +5,209 @@ import './view/css/style.css';
+  todoList.push(todo);
+
+  saveLocal();
+  location.reload(); // eslint-disable-line no-restricted-globals
 }
 
-const saveLocalAndRender = () => {
-  localStorage.setItem('LocalTaskArray', JSON.stringify(myLibrary));
-  render();
+// add button actions
+const btnAddProject = document.getElementById('addProject');
+btnAddProject.onclick = () => { addProject(); };
+
+const btnAddTodo = document.getElementById('addTodo');
+btnAddTodo.onclick = () => { addTodo(projectList[currentProjectIndex].todos); };
+
+const btnEditProject = document.getElementById('editProject');
+btnEditProject.onclick = () => { editProject(btnEditProject.getAttribute('curr-project-index')); };
+
+const btnEditTodo = document.getElementById('editTodo');
+btnEditTodo.onclick = () => { editTodo(btnEditTodo.getAttribute('curr-todo-index')); };
+
+// render todos table
+function renderTodos(project) {
+  currentProjectIndex = projectList.indexOf(project);
+
+  const todoTable = document.getElementById('todo-table');
+  todoTable.innerHTML = '';
+
+  project.todos.forEach((todo) => {
+    const tableRow = todoTable.insertRow();
+    const indexCol = document.createElement('th');
+
+    indexCol.innerHTML = project.todos.indexOf(todo) + 1;
+    tableRow.appendChild(indexCol);
+
+    const titleCol = tableRow.insertCell(1);
+    titleCol.innerHTML = todo.title;
+
+    const dueDateCol = tableRow.insertCell(2);
+    dueDateCol.innerHTML = todo.dueDate;
+
+    const priorityCol = tableRow.insertCell(3);
+    priorityCol.innerHTML = todo.priority;
+    if (todo.priority === 'low') { priorityCol.setAttribute('class', 'badge badge-success'); }
+    if (todo.priority === 'normal') { priorityCol.setAttribute('class', 'badge badge-warning'); }
+    if (todo.priority === 'important') { priorityCol.setAttribute('class', 'badge badge-danger'); }
+
+    const descriptionCol = tableRow.insertCell(4);
+    descriptionCol.innerHTML = todo.description;
+
+    const editCol = tableRow.insertCell(5);
+    const editButton = document.createElement('button');
+    editButton.innerText = 'edit';
+    editButton.classList.add('btn', 'btn-outline-secondary', 'btn-sm');
+    editButton.setAttribute('data-toggle', 'modal');
+    editButton.setAttribute('data-target', '#modalEditTodo');
+    editButton.onclick = () => {
+      setTodoModal(project.todos.indexOf(todo));
+      btnEditTodo.setAttribute('curr-todo-index', project.todos.indexOf(todo));
+    };
+    editCol.appendChild(editButton);
+
+    const removeCol = tableRow.insertCell(6);
+    const removeButton = document.createElement('button');
+    removeButton.innerText = 'remove';
+    removeButton.classList.add('btn', 'btn-outline-danger', 'btn-sm');
+    removeButton.onclick = () => {
+      removeTodo(project, todo);
+    };
+    removeCol.appendChild(removeButton);
+  });
 }
 
-const test = () => {
-  let newProject = project('restaurant page');
+// render projectsList to project table
+function renderProjects(projectList) {
```

Comment:
You ought to use ES6 Syntax for this project:
Which means usage of arrow functions instead of the function keyword.
See this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/es6_requirement.png) from pathwright